src/components/Table/EmptyTable/__tests__/EmptyTable.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import EmptyTable from '../EmptyTable';
+
+const commonTableProps = {
+  id: 'tableid',
+  totalColumns: 3,
+  isFiltered: false,
+};
+
+describe('EmptyTable', () => {
+  it('custom empty state', () => {
+    const { queryAllByText } = render(
+      <EmptyTable {...commonTableProps} emptyState={<span>my custom element</span>} />
+    );
+    expect(queryAllByText('my custom element')).toHaveLength(1);
```

Comment:
Using `screen` cleans things up a bit! [Additional context from Kent's blog](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)

```suggestion
    render(<EmptyTable {...commonTableProps} emptyState={<span>my custom element</span>} />);
    expect(screen.queryAllByText('my custom element')).toHaveLength(1);
```